crates/host/src/wasmbus/handler.rs

```diff
@@ -107,65 +107,113 @@ impl Handler {
 
     #[instrument(level = "trace", skip(self))]
     async fn wrpc_blobstore_blobstore(&self) -> anyhow::Result<wasmcloud_core::wrpc::Client> {
+        let (ns, pkg, iface) = ("wasi", "blobstore", "blobstore");
         let lit = self
             .identify_wrpc_target(&CallTargetInterface::from_parts((
-                "wasi",
-                "blobstore",
-                "blobstore",
+                ns, pkg, iface,
             )))
             .await
-            .context("unknown `wasi:blobstore/blobstore` target")?;
+            .with_context( || {
+                let mut msg = format!("failed to call interface `{ns}:{pkg}/{iface}`");
+                if let Ok(false) = self.try_link_exists_for_interface(ns, pkg, iface) {
+                    msg.push_str(&format!(" (failed to find a configured link from component [{}], please check your configuration)", self.component_id));
+                }
+                msg
+            })?;
         Ok(self.wrpc_client(&lit))
     }
 
     #[instrument(level = "trace", skip(self))]
     async fn wrpc_http_outgoing_handler(&self) -> anyhow::Result<wasmcloud_core::wrpc::Client> {
+        let (ns, pkg, iface) = ("wasi", "http", "outgoing-handler");
         let lit = self
             .identify_wrpc_target(&CallTargetInterface::from_parts((
-                "wasi",
-                "http",
-                "outgoing-handler",
+                ns, pkg, iface
             )))
             .await
-            .context("unknown `wasi:http/outgoing-handler` target")?;
+            .with_context(|| {
+                let mut msg = format!("failed to call interface `{ns}:{pkg}/{iface}`");
+                if let Ok(false) = self.try_link_exists_for_interface(ns, pkg, iface) {
+                    msg.push_str(&format!(" (failed to find a link from this component [{}], please check your configuration)", self.component_id));
```

Comment:
```suggestion
                    msg.push_str(&format!(" (failed to find a configured link from component [{}], please check your configuration)", self.component_id));
```

consistency with other messages suggestion

Reply:
Ah thank you!